app/persistence/valkyrie/persistence/fedora/dynamic_klass.rb

```diff
@@ -2,13 +2,13 @@
 module Valkyrie::Persistence::Fedora
   class DynamicKlass
```

Comment:
Not in scope for this PR, but I'd prefer `DynamicClass` here -- only using `klass` or `clazz` where it would conflict.